module1.js

```diff
@@ -111,6 +131,12 @@ function question7(array1, array2, extraElement) {
 
 function question8(retailPrice, taxAmount) {
   // TODO:
+  var number = retailPrice.substr(1);
+
+  var targetNumber = parseInt(number);
+  var result = ((targetNumber * taxAmount )/0.20).toString();
+  
+  return `£${result}`;
```

Comment:
If you're going to format a value _into_ a string like this, you don't need to explicitly call `toString`.